src/platform/common/utils/localize.ts

```diff
@@ -162,7 +183,7 @@ export namespace ExtensionSurveyBanner {
 export namespace DataScience {
     export const installingModule = localize('products.installingModule', 'Installing {0}');
     export const warnWhenSelectingKernelWithUnSupportedPythonVersion = localize(
-        'DataScience.warnWhenSelectingKernelWithUnSupportedPythonVersion',
+        { key: 'DataScience.warnWhenSelectingKernelWithUnSupportedPythonVersion', comment: ['{Locked="kernel"}'] },
         'The version of Python associated with the selected kernel is no longer supported. Please consider selecting a different kernel.'
```

Comment:
I am assuming we want to keep `kernel` as is and not change it from language to language.